.sauce/config1Dimension.yml

```diff
@@ -18,15 +18,20 @@ testcafe:
 # Controls what files are available in the context of a test run (unless explicitly excluded by .sauceignore).
 rootDir: ./
 suites:
-  - name: "One dimensional Tests in Firefox on Windows"
-    browserName: "firefox"
+  - name: "One dimensional Tests in Edge on Windows"
+    browserName: "microsoftedge"
     src:
       - "Tests/IntegrationTests/Fixtures/1Dimension/*.e2e.js"
     platformName: "Windows 10"
     screenResolution: "1280x1024"
-  - name: "One dimensional Tests in Firefox on MacOS"
-    # todo use chrome here and fix ci https://github.com/neos/neos-ui/issues/3591
+  - name: "One dimensional Tests on Firefox on Windows"
     browserName: "firefox"
+    src:
+      - "Tests/IntegrationTests/Fixtures/1Dimension/*.e2e.js"
+    platformName: "Windows 10"
+    screenResolution: "1280x1024"
+  - name: "One dimensional Tests in Edge on MacOS"
+    browserName: "microsoftedge"
```

Comment:
why not chrome? ^^

Reply:
It was just a try, in safari we also have issues in the tests ;)